services/httpservice/httpservice.go

```diff
@@ -33,6 +34,10 @@ type HTTPServiceImpl struct {
 	RequestTimeout time.Duration
 }
 
+func splitFields(c rune) bool {
+	return unicode.IsSpace(c) || c == ','
```

Comment:
What happens if someone uses a combination of commas and whitespace like `localhost, some-address`? Does `FieldsFunc` return empty strings if you set the setting to something with a comma and white space like `localhost, some-server`?

Reply:
It will discard any combination of the two, so it will just keep the actual hostnames. Eg. https://play.golang.org/p/-CL24ipbg33

Reply:
Excellent. I didn't know `FieldsFunc` works that way, but I guess it would be pretty useless if it didn't